app/models/group.js

```diff
@@ -0,0 +1,22 @@
+var mongoose = require( 'mongoose' );
+var Schema = mongoose.Schema;
+
+// This is the adress schema
```

Comment:
@ivokaragyozov @jordan-enev 
It's very good to have comments around the code, describing the business logic behind specific code, explaining a complex piece of code that cannot be simplified for readability, guiding future usage of a specific functionality and etc. Sometimes however it's also very easy to go over the top with it :)
So in this case `This is the address schema` is mostly clear from the next line ( `var addressSchema = ..` ), another example would be something that I find myself doing very often `// returns the result` followed by `return result;` :laughing: 
These comments may be omitted or be replaced with an explanation of, for example, what is the address schema, what would be contained inside of it, where it would be used, in what occasion it would be good to be extended or reused and something like that.
This will come with experience, but the most proper question to ask yourself when writing a comment would be - if I were to read this a couple of months later what would I need to know about it and how can I remind myself about it in the comment?
Don't feel obligated to change this, but let me know what you think about it and get creative with your comments :rocket: